cmd/shipperctl/config/kubeconfig.go

```diff
@@ -25,3 +26,15 @@ func Load(kubeConfigFile, managementClusterContext string) (kubernetes.Interface
 	}
 	return kubeClient, shipperClient, nil
 }
+func LoadInClusterConfig(restConfig *rest.Config) (kubernetes.Interface, shipperclientset.Interface, error) {
+	kubeClient, err := configurator.NewKubeClientFromInClusterKubeConfig(restConfig)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	shipperClient, err := configurator.NewShipperClientFromInClusterKubeConfig(restConfig)
+	if err != nil {
+		return nil, nil, err
+	}
```

Comment:
Why not using `NewClusterConfigurator(restConfig)` directly here and delete `NewKubeClientFromInClusterKubeConfig` and `NewShipperClientFromInClusterKubeConfig`?

Reply:
Actually, even removing this function and just use `NewClusterConfigurator` in the upper scope in place of `LoadInClusterConfig`.